packages/@okta/vuepress-site/guides/sign-into-mobile-app/stay-signed-in/android/refresh.md

```diff
@@ -0,0 +1,15 @@
+Use the `refreshToken` method to get a new access token:
```

Comment:
@FeiChen-okta How will the developer know they need to do this? Wait for a 401 response from their API, or proactively check?

Reply:
I'll add example code on how to check. The Token class contains a method that checks expiration.